docs/Reference/Release-Types.md

```diff
@@ -0,0 +1,16 @@
+---
+title: QKM release types
+---
+
+# Release types
+
+Quorum Key Manager has 2 different release types - long term support (LTS) releases and interim releases.
+
+LTS releases are reliable, production-ready releases that are updated and supported throughout the lifetime
+of the release. LTS releases are supported for 12 months and are patched with every bug fix during the
+course of its support timeline. LTS releases use the format `v<year>.<month>.0`, for example v21.12.0, and are
```

Comment:
There is not different in the format of LTS and non LTS releases, so I would rather skip that here and just mention LTS version are defined in the CHANGELOG

Reply:
Fixed